.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 *.sw?
 .DS_Store
 runtime.json
+.idea
```

Comment:
it's a slippery slope of adding all files ever? I'd prefer if this was added to your `~/.gitignore` which you can customize with whatever you personally are running. (enable this via `git config --global core.excludesfile ~/.gitignore`)